src/supermarket/app/views/cookbooks/directory.html.erb

```diff
@@ -61,13 +61,13 @@
       <h3>How do I use a cookbook?</h3>
       <p>Interact with cookbooks by using the <code>knife cookbook</code> commands.
       With <code>knife cookbook</code>, you can create, delete, show, list,
-      download and upload cookbooks. Read the <%= link_to 'knife cookbook commands documentation', chef_docs_url('knife_cookbook.html') %> for more information.</p>
+      download and upload cookbooks. Read the <%= link_to 'knife cookbook commands documentation', chef_docs_url('knife_cookbook') %> for more information.</p>
```

Comment:
These are going to need hand verification to check the new URL. This one isn't docs.chef.io/knife_cookbook. It's https://docs.chef.io/workstation/knife_cookbook/

Reply:
Yes, will change the same